frontend/src/routes/__root.tsx

```diff
@@ -4,18 +4,142 @@ import { Logo } from "~/components/logo";
 import { MetaTitle } from "~/components/meta-title";
 import { TailwindIndicator } from "~/components/tailwind-indicator";
 import { Button } from "~/components/ui/button";
+import {
+  Menubar,
+  MenubarContent,
+  MenubarItem,
+  MenubarMenu,
+  MenubarTrigger
+} from "~/components/ui/menubar";
+
+import {
+  AlarmCheck,
+  BookOpen,
+  ChevronDown,
+  ChevronsUpDown,
+  CircleUser,
+  Folder,
+  Globe,
+  Hammer,
+  HeartHandshake,
+  HelpCircle,
+  LogOut,
+  Search,
+  Send,
+  Settings,
+  Twitter
+} from "lucide-react";
+import { useAuthUser } from "~/components/helper/use-auth-user";
+import { Input } from "~/components/ui/input";
 
 export const Route = createRootRoute({
   component: () => (
     <main className="bg-background">
+      <Navigation />
       <Outlet />
-      <TailwindIndicator />
-      <TanStackRouterDevtools />
+      <Footer />
     </main>
   ),
   notFoundComponent: NotFound
 });
 
+function Navigation() {
+  const query = useAuthUser();
+  const user = query.data?.data?.user;
+  if (!user) {
+    return null;
+  }
+  return (
+    <>
+      <header className="flex px-6 border-b border-opacity-65 border-border py-2 items-center bg-toogle t justify-between gap-4">
+        <Logo className="w-10 flex-none h-10 mr-8" />
+        <div className="flex  w-full items-center">
+          <Menubar className="border-none w-fit text-black bg-primary">
+            <MenubarMenu>
+              <MenubarTrigger className="flex justify-center text-sm items-center gap-1">
+                Create
+                <ChevronsUpDown className="w-4" />
+              </MenubarTrigger>
+              <MenubarContent className="border min-w-6 border-border">
+                <MenubarItem className="flex gap-5">
+                  <Folder className="w-4 opacity-50" />
+                  Project
+                </MenubarItem>
+                <MenubarItem className="flex gap-5">
+                  <Globe className="w-4 opacity-50" />
+                  Web Service
+                </MenubarItem>
+                <MenubarItem className="flex gap-5">
+                  <Hammer className="w-4 opacity-50" />
+                  Worker
+                </MenubarItem>
+                <MenubarItem className="flex gap-5">
+                  <AlarmCheck className="w-4 opacity-50" />
+                  CRON
+                </MenubarItem>
+              </MenubarContent>
+            </MenubarMenu>
+          </Menubar>
```

Comment:
Refactor this into a component so that you don't have to repeat the classes twice. 
Even if you don't intend to use the component somewhere else outside of here, make a component in the same file, This menubar content (and style) may be changed in the future, and we want it to have the same visual style for all the menus.